packages/logging/src/setup-logger-and-telemetry.ts

```diff
@@ -42,7 +42,7 @@ class MultiSet<T> {
   _entries: Map<string, number> = new Map();
 
   add(entry: T): void {
-    const key = JSON.stringify(Object.entries(entry).sort());
+    const key = JSON.stringify(Object.entries(entry as Record<string, any>).sort());
```

Comment:
Mh, this one is a valid complaint – this line should probably be left as-is and instead the class definition should read `class MultiSet<T extends Record<string, any>> {`